### DISCUSSION TITLE
离线数据的问题

### DISCUSSION THREAD

**Discussion by yangGuangChaoGit** [zouyaoji/vue-cesium#553] [Q&A]
求助大佬，我使用vue3去引入组件库，想使用离线的地球，配置了cesiumPath，地址也可以通过http访问到cesium文件，但是组件库中还是调用了默认的https://unpkg.com/cesium@latest/Build/Cesium/Cesium.js路径，请问是我哪里缺少配置了吗，我使用的是3.2.7的版本。

**Comment by yangGuangChaoGit** [comment]
已经解决了

**Reply by ShanShan1214** [reply]
> 已经

怎么解决的啊 同问
